src/main/resources/application.yml

```diff
@@ -2,6 +2,7 @@
 ---
 "%test":
   ide-sidecar:
+    schema-fetch-error-ttl: 30
```

Comment:
Nit: can we put this lower, maybe just before `schema-fetch-retry` (since before it would be in proper sort-order)?